core/lensUtility.ml

```diff
@@ -0,0 +1,134 @@
+open Utility
+
+(* Lenses *)
+module ColSet = struct
```

Comment:
ColSet seems like a very small delta over StringSet.  Would it make sense to just add this pretty-printing function to Utility.StringSet?